MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -55,15 +57,40 @@ def store_issue_details(issue, repo, db):
 	issue_details['created_at'] = issue['created_at']
 	issue_details['updated_at'] = issue['updated_at']
 	issue_details['labels'] = issue['labels']
-	
+
 	issue_details['issue_content_path'] = issue_content_path
 	# issue_details['test_output_path'] = test_output_path
 	issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+	trss_jenkins_output_path = f'./data/GitHubData/JenkinsOutput/{issue_uid}'
 	#Store issue content at issue_content_path
 	store_on_fs(issue['body'], issue_content_path)
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	content_extracted = extract_jenkins_link_and_testname(issue['body'])
+	issue_quoted_jenkins_links = content_extracted[0]
+	issue_quoted_test_names = content_extracted[1]
+	if issue_quoted_jenkins_links != []:
+    		issue_details['issue_quoted_jenkins_links'] = issue_quoted_jenkins_links
+	if issue_quoted_test_names != []:
+    		issue_details['issue_quoted_test_names'] = issue_quoted_test_names
+
+	if not (jenkins_links == [] and test_names == []):
+		for link in jenkins_links:
+		    for test in test_names:
+			new_string = link.split('/')
+			final_string = ''.join(new_string)
+			issue_uid2 = f'{repo_name}_{issue_number}_{final_string}_{test}.txt'
+			jenkins_output_path = f'./data/GitHubData/JenkinsOutput/{issue_uid2}'
+			store_on_fs(content_extracted, jenkins_output_path)
+
+			if not os.path.isfile(issue_uid2):
+			    trss_jenkins_output = query_trss_for_jenkins_output(link, test)
+			    if trss_jenkins_output != '':
+				store_on_fs(trss_jenkins_output, jenkins_output_path)	
+
```

Comment:
After save in file, the `issue_details` with this path should also be added. Since multiple paths may be added, save it as a list.